dicom_utils/tags.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from typing import Final, Iterable, Tuple, Union
+
+from dicomanonymizer.dicomfields import ALL_TAGS
+from pydicom.tag import Tag as PydicomTag
+
+from ._tag_enum import Tag
+
+
+def create_tag(val: Union[str, int, Tuple[int, int]]) -> Tag:
+    r"""Create a tag from a string keyword or int value"""
+    if isinstance(val, str):
+        try:
+            return getattr(Tag, val)
+        except AttributeError:
+            raise ValueError(f"Invalid tag {val}")
+    elif isinstance(val, int):
+        return Tag(val)
+    elif isinstance(val, tuple):
+        return Tag(PydicomTag(*val))
+    else:
+        raise TypeError(f"Expected int, str or 2-tuple - found {type(val)}")
+
+
+def get_display_width(tags: Iterable[Tag]) -> int:
+    r"""Returns the width of the longest tag string"""
+    return max((len(str(tag)) for tag in tags), default=0)
+
+
+# Add PHI tags based on keyword
+PHITags: Final = {create_tag(t) for t in ALL_TAGS if len(t) == 2}
```

Comment:
Maybe declare type as `Final[Set[Tag]]`?

Reply:
My development environment detects `PHITags` as `Set[Tag]` when declared as `Final`. I think declaring as `Final` is sufficient to resolve `Set[Tag]`. We can change it but I don't think it's necessary.